src/components/common/FilterByCategory/FilterByCategory.js

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styles from './FilterByCategory.module.scss';
+
+const FilterByCategory = ({ numberOfProductsByCategory, activeCategory }) => {
+  return (
+    <div className={styles.wrapper}>
+      <p>
+        <span className={styles.title}>FILTER BY CATEGORIES</span>
+      </p>
+      <div className={styles.borderLine}>
+        {numberOfProductsByCategory.map(({ name, quantity, id }, index) => (
+          <div key={index}>
+            {activeCategory === id ? (
+              <p className={styles.textWrapper}>
+                <span className={styles.activeText}>&gt;&nbsp;&nbsp;&nbsp;{name}</span>
```

Comment:
Dlaczego użyłeś `&gt;&nbsp;&nbsp;&nbsp;` , mamy w projekcie FontAwsome które można do tego wykorzystać

Reply:
Pomyślałem, że jeśli można wyświetlić element bez posiłkowania się ściąganiem elementów z zewnątrz to może choć troche wpłynie to na szybkość ładowania strony... mam zmienić na element z FontAwesome?

Reply:
To nie pogorszy niczego, a w dodatku mamy w projekcie FontAwsome używane daltego Ci o tym napisałem